core/embed/rust/src/ui/component/text/layout.rs

```diff
@@ -29,6 +29,13 @@ pub struct TextLayout {
     /// Bounding box restricting the layout dimensions.
     pub bounds: Rect,
 
+    /// Additional space before beginning of text, can be negative to shift text
+    /// upwards.
+    pub top_space: i32,
+    /// Additional space between end of text and bottom of bounding box, can be
+    /// negative.
+    pub bottom_space: i32,
+
```

Comment:
Personally I would call these `padding_top` and `padding_bottom`.

Reply:
Or maybe we should be using `Insets` for this?

Reply:
The difference to just insetting `bounds` is that the returned text height includes the padding, right?

Reply:
Renamed the fields in 6dd100c.